pages/student/06-writing_a_proposal.md

```diff
@@ -48,7 +48,7 @@ Include a brief, clear work breakdown structure with milestones and deadlines. M
 
 **Related Work**
 
-You should understand and communicate other people's work that may be related to your own. Do your research, and make sure you understand how the project your are proposing fits into the target organization. Be sure to explain how the proposed work is different from similar related work.
+You should understand and communicate other people's work that may be related to your own. Do your research, and make sure you understand how the project you are proposing fits into the target organization. Be sure to explain how the proposed work is different from similar related work.
```

Comment:
Looks like there is one more occurrence of this typo :)

Can you fix all of them here? Thanks!

https://github.com/google/gsocguides/search?q=%22your+are%22&unscoped_q=%22your+are%22